### PR TITLE
ADR-1321: Updated UI tests with new spoilt journey changes

### PR DESCRIPTION
**Pull request by Kavitakum**

### CODE REVIEW

File: src/test/scala/uk/gov/hmrc/alcoholDuty/cucumber/runner/Runner.scala [hunk 1]
```diff
@@ -25,7 +25,7 @@ import org.junit.runner.RunWith
   features = Array("src/test/resources/features"),
   glue = Array("uk.gov.hmrc.alcoholDuty.cucumber.stepdefs"),
   plugin = Array("pretty", "html:target/cucumber", "json:target/cucumber.json", "junit:target/test-reports/Runner.xml"),
-  tags = "@Test"
+  tags = "@check"
```
Comment (chiransan01) [inline src/test/scala/uk/gov/hmrc/alcoholDuty/cucumber/runner/Runner.scala:28]: Revert back to @Test

File: src/test/scala/uk/gov/hmrc/alcoholDuty/pages/alcoholDuty/adjustments/SpoiltAlcoholTypePage.scala [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+package uk.gov.hmrc.alcoholDuty.pages.alcoholDuty.adjustments
+
+import uk.gov.hmrc.alcoholDuty.conf.TestConfiguration
+import uk.gov.hmrc.alcoholDuty.pages.BasePage
+
+object SpoiltAlcoholTypePage extends BasePage {
+
+  override val url: String    = TestConfiguration.url("alcohol-duty-returns-frontend") + "/spoilt-alcohol-type"
+  override val newUrl: String =
+    TestConfiguration.url("alcohol-duty-returns-frontend") + "/change-spoilt-alcohol-type"
```
Comment (chiransan01) [inline src/test/scala/uk/gov/hmrc/alcoholDuty/pages/alcoholDuty/adjustments/SpoiltAlcoholTypePage.scala:26]: Can you check with Del to make sure that these are the correct URLs as that does not seem like fit with existing URLs

File: src/test/scala/uk/gov/hmrc/alcoholDuty/pages/alcoholDuty/adjustments/SpoiltAlcoholVolumePage.scala [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+ */
+
+package uk.gov.hmrc.alcoholDuty.pages.alcoholDuty.adjustments
+
+import uk.gov.hmrc.alcoholDuty.conf.TestConfiguration
+import uk.gov.hmrc.alcoholDuty.pages.BasePage
+
+object SpoiltAlcoholVolumePage extends BasePage {
+
+  override val url: String    = TestConfiguration.url("alcohol-duty-returns-frontend") + "/spoilt-alcohol"
```
Comment (chiransan01) [inline src/test/scala/uk/gov/hmrc/alcoholDuty/pages/alcoholDuty/adjustments/SpoiltAlcoholVolumePage.scala:24]: Same here, check URLs please

File: src/test/scala/uk/gov/hmrc/alcoholDuty/pages/alcoholDuty/adjustments/SpoiltBeerAlcoholVolumePage.scala [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+ */
+
+package uk.gov.hmrc.alcoholDuty.pages.alcoholDuty.adjustments
+
+import uk.gov.hmrc.alcoholDuty.conf.TestConfiguration
+import uk.gov.hmrc.alcoholDuty.pages.BasePage
+
+object SpoiltBeerAlcoholVolumePage extends BasePage {
+
+  override val url: String    = TestConfiguration.url("alcohol-duty-returns-frontend") + "/spoilt-alcohol"
```
Comment (chiransan01) [inline src/test/scala/uk/gov/hmrc/alcoholDuty/pages/alcoholDuty/adjustments/SpoiltBeerAlcoholVolumePage.scala:24]: Same here, check URLs

File: src/test/scala/uk/gov/hmrc/alcoholDuty/cucumber/runner/Runner.scala (hunk 1 above)
Comment (Kavitakum) [reply src/test/scala/uk/gov/hmrc/alcoholDuty/cucumber/runner/Runner.scala]: Updated

File: src/test/scala/uk/gov/hmrc/alcoholDuty/pages/alcoholDuty/adjustments/SpoiltAlcoholVolumePage.scala (hunk 3 above)
Comment (Kavitakum) [reply src/test/scala/uk/gov/hmrc/alcoholDuty/pages/alcoholDuty/adjustments/SpoiltAlcoholVolumePage.scala:24]: There is communication as of now about new urls for Spoilt. Sinimol has posted question to Del and Yash

File: src/test/scala/uk/gov/hmrc/alcoholDuty/pages/alcoholDuty/adjustments/SpoiltBeerAlcoholVolumePage.scala (hunk 4 above)
Comment (Kavitakum) [reply src/test/scala/uk/gov/hmrc/alcoholDuty/pages/alcoholDuty/adjustments/SpoiltBeerAlcoholVolumePage.scala:24]: There is communication as of now about new urls for Spoilt. Sinimol has posted question to Del and Yash

Comment (chiransan01) [reply src/test/scala/uk/gov/hmrc/alcoholDuty/pages/alcoholDuty/adjustments/SpoiltBeerAlcoholVolumePage.scala:24]: Del has updated the new URLs so this needs to be added

File: src/test/scala/uk/gov/hmrc/alcoholDuty/pages/alcoholDuty/adjustments/SpoiltAlcoholVolumePage.scala (hunk 3 above)
Comment (chiransan01) [reply src/test/scala/uk/gov/hmrc/alcoholDuty/pages/alcoholDuty/adjustments/SpoiltAlcoholVolumePage.scala:24]: Del has updated the new URLs so this needs to be added

File: src/test/scala/uk/gov/hmrc/alcoholDuty/pages/alcoholDuty/adjustments/SpoiltAlcoholTypePage.scala (hunk 2 above)
Comment (chiransan01) [reply src/test/scala/uk/gov/hmrc/alcoholDuty/pages/alcoholDuty/adjustments/SpoiltAlcoholTypePage.scala:26]: Del has updated the new URLs so this needs to be added

**Review by chiransan01 — APPROVED** [review]

**Review by chiransan01 — APPROVED** [review]

**Pull request merged**